src/app/components/game-components/game-component/GameComponent.spec.ts

```diff
@@ -145,7 +167,7 @@ describe('GameComponent', () => {
                 TestBed.createComponent(gameInfo.component).debugElement.componentInstance;
             expect(gameComponent.encoder).withContext('Encoder missing for ' + gameInfo.urlName).toBeTruthy();
             expect(gameComponent.tutorial).withContext('tutorial missing for ' + gameInfo.urlName).toBeTruthy();
-            if (gameComponent.tutorial) {
+            if (gameComponent.tutorial != null) {
```

Comment:
should perhaps be coded as if tutorial are never null (and, make it happend, we don't have gameComponent without tutorial)...
Faudrait juste changer la façon dont on crée le composant, un super(blblblblblblb) très rempli :D

Reply:
Condition enlevée car toujours vraie effectivement.
Pour le super, une façon peut-être plus jolie serait avec `strictPropertyInitialization`, qui te dirait simplement « tiens t'as un champ public que t'initialise pas dans ton constructeur, mange tes morts je compile pas cette merde » (donc pas besoin de super super)